.github/workflows/release-publish.yml

```
@@ -137,6 +137,32 @@ jobs:
+                -o /tmp/changelog.md
+
+              CHANGELOG=$(cat /tmp/changelog.md)
+              chmod +x .github/scripts/close-jira-tickets.sh
+              .github/scripts/close-jira-tickets.sh \
+                -t "$TAG" \
+                -d "$CHANGELOG"
+
+              echo "Jira tickets closed"
+
+
+
```

YAMLlint (1.38.0)

[error] too many blank lines (3 > 2)

(empty-lines)